pyroomacoustics/libroom_src/libroom.cpp

```diff
@@ -30,6 +34,10 @@ Room *create_room(py::list _walls, py::list _obstructing_walls, const Eigen::Mat
     room->obstructing_walls.push_back(owall.cast<int>());
 
   room->dim = room->walls[0].dim;
+  
+  // Specific to ray tracing
+  room->mic_pos = _microphones.col(0);
```

Comment:
Hum, it would be best if the ray tracing can also handle multiple microphones. In that case, you would need to add a loop that checks for intersection with all the microphones within the ray tracer.